### DISCUSSION TITLE
ximi

### DISCUSSION THREAD

**Discussion by nosebs** [zenfyrdev/bootloader-unlock-wall-of-shame#3] [General]
https://gsmchina.com/xiaomi-will-no-longer-allow-bootloader-unlocking-5618/
https://xiaomiui.net/xiaomi-bootloader-lock-will-now-be-unlocked-in-a-new-way-52312/
https://web.vip.miui.com/page/info/mio/mio/detail?postId=41559680&fromPathname=mioHomePage&app_version=dev.231026&ref=share

**Comment by zenfyrdev** [comment]
Huh, thanks for the info. I guess my concerns about ximi were correct after all. 

Btw, it's pretty funny how these articles downplay this situation. "Xiaomi bootloader lock will now be unlocked in a new way" - making it as obnoxious as possible is surely a "new way". No way this was written by a "No.1 Xiaomi Fan Community".

**Comment by nosebs** [comment]
https://xiaomiui.net/say-goodbye-to-hyperos-global-bootloader-unlock-52670/

waiting for official accouterment...
